packages/digestif/digestif.0.7.4/opam

```diff
@@ -47,7 +47,6 @@ depopts: [
 conflicts: [
   "mirage-xen-posix" {< "3.1.0"}
   "ocaml-freestanding" {< "0.4.1"}
-  "mirage-runtime"{< "4.0.0"}
```

Comment:
shouldn't it be `"mirage-runtime" {>= "4.0.0"}` instead then? or is 0.7.4 working with both mirage3 and mirage4?

Reply:
Hmmhmm, my workspace becomes some complex, I will follow your advise :+1: and be conservative.